src/main/java/org/jenkinsci/plugins/argusnotifier/JenkinsRunFormatter.java

```diff
@@ -6,11 +6,13 @@
 
 import javax.annotation.Nonnull;
 
+
 /**
  * Formatter to consistently format Jenkins and Jenkins build info
  */
 class JenkinsRunFormatter {
     
+	public static final String GIT_COMMIT = "GIT_COMMIT";
```

Comment:
Can you please format all of the source files that you have modified? Looks like there is mismatch with indentation settings.

Reply:
fixed

Reply:
Use the IDE formatter to replaced TAB with spaces